src/services/hall.services/updateHall.ts

```diff
@@ -0,0 +1,23 @@
+import { NextFunction } from 'express';
+
+import { IHall } from '../../interfaces';
+import { HallModel } from '../../models/hall';
+import { CustomError } from '../../utils/response/custom-error/customError';
+
+export const updateHallService = async (
+  id: string,
+  payload: IHall,
+  next: NextFunction,
+): Promise<void | IHall> => {
+  try {
+    const existingHall = await HallModel.findById({ _id: id });
+    if (!existingHall) {
+      return next(new CustomError(400, 'General', "Hall ID doesn't exist!"));
+    }
+    // console.log(payload)
```

Comment:
what is this comment for?

Reply:
Sorry I'll clear that, I used the console log